### PR TITLE
chore(mobileapp): Fix iPad layout

### LINKED ISSUE #6966: iOS header overlaps with UI
**Describe the bug**
The iOS header (time, date, battery etc..) overlaps with the UI (see top right of the screen).

**To Reproduce**
Open app

**Expected behavior**
iOS header should not overlap

**Screenshots**
Screenshot 2024-07-09 at 14 15 47 jpeg

**Additional context**
Using the app on iPad (classic)

### PR DESCRIPTION
**Pull request by VIKTORVAV99**
## Issue

Fixes #6966 

## Description

Ensures that the safe areas are respected in the relevant parts of the app.

### Preview

image

### Double check

- [x] I have run `pnpx prettier@2 --write .` and `poetry run format` in the top level directory to format my changes.

### CODE REVIEW

**Comment by corradio** [comment]
Super cool! I'll remove myself from the review as I don't feel I have the background required to review this.
By the way, would we consider hiding the left panel in portrait mode? It would feel more intuitive to me, but maybe that's a different issue!

**Comment by VIKTORVAV99** [comment]
> Super cool! I'll remove myself from the review as I don't feel I have the background required to review this. By the way, would we consider hiding the left panel in portrait mode? It would feel more intuitive to me, but maybe that's a different issue!

Different issue but yeah we can consider hiding it by default in portrait mode!

Perfectly fine with the review 👍🏻

**Comment by cadeban** [comment]
Tested via xcode simulator and it looks good in both orientations:
Screenshot 2024-11-14 at 09 50 40
Screenshot 2024-11-14 at 09 50 23

**Comment by VIKTORVAV99** [comment]
@cadeban mind pressing that green approval button is it looks good? 👀 💚

File: web/src/features/map-controls/MapControls.tsx [hunk 1]
```diff
@@ -116,7 +116,7 @@ function DesktopMapControls() {
   const isConsumptionOnlyMode = useFeatureFlag('consumption-only');
 
   return (
-    <div className="pointer-events-none absolute right-3 top-2 z-20 hidden flex-col items-end md:flex">
+    <div className="pointer-events-none absolute right-3 top-2 z-20 mt-[env(safe-area-inset-top)] hidden flex-col items-end md:flex">
```
Comment (cadeban) [inline web/src/features/map-controls/MapControls.tsx:119]: nit: Should we consider adding a fallback value for all instances of `env(safe-area-inset-top)`?

Comment (VIKTORVAV99) [reply web/src/features/map-controls/MapControls.tsx:119]: Not sure we need to, I only added it where we do a calc to be on the safe side.

But I can do it if we want to, previously we didn't have any though (when it was just for mobile).

**Review by cadeban — APPROVED** [review]
lgtm!

**Pull request merged**